modules/standard/IO.chpl

```diff
@@ -4650,7 +4652,7 @@ proc _channel.readLine(ref a: [] ?t, maxSize=a.size, stripNewline=false): int th
         // encountered an error so throw
         this._revert();
         var err:errorCode = -got;
-        try this._ch_ioerror(EFORMAT:errorCode, "in channel.readLine(a : [] uint(8))");
+        try this._ch_ioerror(err, "in channel.readLine(a : [] uint(8))");
```

Comment:
It looks this change was only made in this overload of `readLine`. Is there a reason this one uses the value of `err`, but the other overloads still construct the error with `EFORMAT:errorCode`?

Reply:
This was the only overload that used `EFORMAT:errorCode` in this position. All of the overloads use it further down, when erroring for an overly-long line, because there is no error code coming from the runtime to use there. My change just makes them all consistent as far as I can tell.

Reply:
Ok, I see. I didn't read the other overloads closely enough. Thanks!